sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptionsValidator.java

```diff
@@ -46,6 +46,26 @@
    * @return The type
    */
   public static <T extends PipelineOptions> T validate(Class<T> klass, PipelineOptions options) {
+    return validate(klass, options, false);
+  }
+
+  /**
+   * Validates that the passed {@link PipelineOptions} from command line interface (CLI)
+   * conforms to all the validation criteria from the passed in interface.
+   *
+   * <p>Note that the interface requested must conform to the validation criteria specified on
+   * {@link PipelineOptions#as(Class)}.
+   *
+   * @param klass The interface to fetch validation criteria from.
+   * @param options The {@link PipelineOptions} to validate.
+   * @return The type
```

Comment:
Here and above.
nit: "The type" -> `Validated options.`

I'll fix this while merging.